pyk/src/pyk/integration_tests/test_emit_json_spec.py

```diff
@@ -3,7 +3,7 @@
 
 from ..kast import EMPTY_ATT, KAst, KDefinition, KFlatModuleList, KRequire
 from ..kastManip import remove_generated_cells
-from ..ktool import KompileBackend, kprove
+from ..ktool import KompileBackend, _kprove
```

Comment:
```suggestion
from ..ktool.kompile import KompileBackend, _kprove
```

Reply:
I'm not sure this works, I tried it, but will try again.

Reply:
Oh, why wouldn't it work?

Reply:
I don't know, probably cause I'm bad at computers. I've updated it with these `_` imports hidden and made more explicit where needed.